scripts/bootloader/provision.py

```diff
@@ -153,8 +154,13 @@ def get_hashes(public_key_files, verify_hashes):
 
 
 def main():
+    global lcs_state_size
```

Comment:
I think this should be refactored to not be a global variable. Making it a parameter to `generate_mcuboot_only_provision_hex_file` should be enough.